PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+## Description
+
+*What does this PR do?*
+
+#### Issue Addressed (if applicable)
+
+*[Link issue here]()*
+
+#### Before/After Screenshots (if applicable)
+
+*Insert images here*
+
+
+## Steps to Test
+
+- [ ] *Step 1*
+- [ ] *Step 2*
+
+## Implementation Notes
```

Comment:
I'd say this field can be optional too if the tech implementation is not complicated.